ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1195,3 +1195,59 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  hidden: true
+  interval: 12h
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-ci-secret-bootstrap
+  spec:
+    containers:
+    - args:
+      - --bw-password-path=/etc/ci-secret-bootstrap/bw_password
+      - --bw-user=tbd
+      - --config=./core-services/ci-secret-bootstrap/_config.yaml
+      - --dry-run=false
+      - --force=true
+      command:
+      #- /usr/bin/ci-secret-bootstrap
+      # better to see it is hidden on deck first
+      - echo
+      env:
+      - name: KUBECONFIG
+        value: /etc/build-farm-credentials/sa.config-updater.api.ci.config:/etc/build-farm-credentials/sa.config-updater.app.ci.config:/etc/build-farm-credentials/sa.config-updater.build01.config
+      image: registry.svc.ci.openshift.org/ci/ci-secret-bootstrap:latest
+      imagePullPolicy: Always
+      name: ci-secret-bootstrap
+      resources:
+        requests:
+          cpu: 500m
+      volumeMounts:
+      - mountPath: /etc/ci-secret-bootstrap
+        name: ci-secret-bootstrap
+        readOnly: true
+      - mountPath: /etc/build-farm-credentials
+        name: build-farm-credentials
+        readOnly: true
+    volumes:
+    - name: ci-secret-bootstrap
+      secret:
+        secretName: bitwarden-credentials-OpenShift-CI-Bitwarden-Bot
+    - name: build-farm-credentials
+      secret:
+        items:
+        - key: sa.config-updater.api.ci.config
+          path: sa.config-updater.api.ci.config
+        - key: sa.config-updater.app.ci.config
+          path: sa.config-updater.app.ci.config
```

Comment:
Seems a bit bizzare to mount this in and not just set the service account for the pod but that might be harder to union

Reply:
The unioning is the one thing, the other is the identification by context name :)

Reply:
Context is the tricky part. True.
InClusterConfig ... it would make the implementation even harder.